lib/shopify-cli/project.rb

```diff
@@ -130,6 +130,11 @@ def __directory(curr)
     #
     #   ShopifyCli::Project.current.env
     #
+
+    def clear_env
```

Comment:
I think this splits up the env method and its documentation, could we move this before or after it?